src/main/java/com/eerussianguy/blazemap/feature/maps/WorldMapGui.java

```diff
@@ -283,6 +285,26 @@ public void render(GuiGraphics graphics, int i0, int i1, float f0) {
         }
     }
 
+    private void renderExportProgress(GuiGraphics graphics, float scale) {
+        AtlasExporter.Task task = AtlasExporter.getTask();
+        if(task == null) return;
+        Font font = Minecraft.getInstance().font;
+        var stack = graphics.pose();
+        stack.pushPose();
+        stack.translate(width - 205, 5, 0);
+        RenderHelper.fillRect(stack.last().pose(), 200, 30, Colors.WIDGET_BACKGROUND);
+        int total = task.getTilesTotal();
+        int current = task.getTilesCurrent();
+        double progress = ((double)current) / ((double)total);
+        String tiles = String.format("%d / %d tiles", current, total);
+        graphics.drawString(font, "Exporting", 5, 5, Colors.WHITE);
+        graphics.drawString(font, tiles, 195 - font.width(tiles), 5, Colors.WHITE);
+        stack.translate(5, 17, 0);
+        RenderHelper.fillRect(stack.last().pose(), 190, 10, Colors.LABEL_COLOR);
+        RenderHelper.fillRect(stack.last().pose(), (int)(190*progress), 10, Colors.WHITE);
+        stack.popPose();
```

Comment:
Nit: Have ya heard of vertical whitespace? 😜

Reply:
No 🙃

Reply:
Added whitespace and comments in working branch